ref/general/mp-openapi.adoc

```diff
@@ -47,7 +57,10 @@ For example, the following code example shows JAX-RS code to request the list of
     }
 ----
 
-In the following OpenAPI documentation example that is generated from the example code, the JAX-RS framework generates the path /inventory/systems path based on the value of the @Path annotation, and the value of the operationId key (listContents) based on the name of the method (listContents()). The other parts of the document are generated from the OpenAPI annotations that the developer annotated the code with. For example, the @Operation annotation provides more meaningful information about what the operation does and why; in this case, the operation returns a list of the host names with properties of each host that are currently stored by the inventory web service. This is described by the summary and description keys in the @Operation annotation:
+In the following OpenAPI documentation example that is generated from the example code, the JAX-RS framework generates the `path /inventory/systems` path based on the value of the `@Path` annotation, and the value of the `operationId` key (listContents) based on the name of the method (listContents()).
```

Comment:
With the phrase "that is generated from the example code", I think maybe replace "the example code" with a reference to what the example code is so if the example code is a JAX-RS request then change it to something like "that is generated from the JAX-RX request". This can help highlight exactly what the user needs to know about the example.